packages/shared/utils.ts

```diff
@@ -23,9 +23,35 @@ export function toNumber(value: string): number | string {
   return isNaN(n) ? value : n;
 }
 
+export function isObjectLike(value: any) {
+  return typeof value === 'object' && value !== null;
+}
+
+export function getTag(value: any) {
+  if (value == null) {
+    return value === undefined ? '[object Undefined]' : '[object Null]';
+  }
+  return Object.prototype.toString.call(value);
+}
+
+export function isPlainObject(value: any) {
```

Comment:
Let's add a link for where this is pulled from, like lodash's src code link. Just to be a reference.

Reply:
Added ✅